lute/templates/backup/index.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% block title %}Backups{% endblock %}
+
+{% block header %}Backups{% endblock %}
+
+{% block body %}
+
+{% if backups|length == 0 %}
+  <p>No backups have been made yet.</p>
+{% else %}
+
+<table class="table dataTable no-footer" style="width: 50%; margin: 0;">
+  <thead>
+    <tr>
+      <th>Backup File</th>
+      <th>Size</th>
+      <th>Last Modified</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for backup in backups %}
+    <tr>
+      <td>{{ backup.name }}</td>
+      <td>{{ backup.size }}</td>
+      <td>{{ backup.last_modified }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
+
+{% endif %}
+
+<br />
+<a href="{{ url_for('backup.backup', type='manual') }}">Create new</a>
```

Comment:
Ah you moved the "create backup" link here.  Let's also keep it in the top-level menu, I like having it readily available.